next-app/src/lib/engine/managers/transition-manager.ts

```
@@ -1,116 +1,355 @@
-  vec4 getToColor(vec2 uv) {
-    return texture2D(uSamplerTo, uv);
+  /**
+   * Reset both sprites to normal state (called when transition ends or is removed).
+   */
+  resetTransition(transition: Transition): void {
+    if (!this._spriteGetter) return
+    const outgoing = this._spriteGetter(transition.outgoing.id)
+    const incoming = this._spriteGetter(transition.incoming.id)
+    if (outgoing) {
+      outgoing.alpha = 1
+      outgoing.x = outgoing.x // keep position (set by updateRect next frame)
```

Biome (2.3.13)

[error] x is assigned to itself.

This is where is assigned.

Self assignments have no effect and can be removed.

(lint/correctness/noSelfAssign)